vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/VimStringParserBase.kt

```diff
@@ -178,7 +182,12 @@ abstract class VimStringParserBase : VimStringParser {
   private fun getMapLeader(): List<KeyStroke> {
     val mapLeader: Any? = injector.variableService.getGlobalVariableValue("mapleader")
     return if (mapLeader is VimString) {
-      stringToKeys(mapLeader.value)
+      val v: String = mapLeader.value
+      if (v[0] == '<' && v[v.length - 1] == '>') {
```

Comment:
We should check for `\<...>`, but only if we let the escape char flow from the parser (see comment below)